lnbits/core/views/admin_api.py

```diff
@@ -125,7 +125,10 @@ async def api_topup_balance(data: CreateTopup) -> dict[str, str]:
 
     await update_wallet_balance(wallet_id=data.id, amount=int(data.amount))
 
-    return {"status": "Success"}
+    return {
+        "status": "Success creating virtual funds."
+        "Payments depend on actual funds on funding source."
```

Comment:
```suggestion
        "Payments are dependent on the actual funds available in the funding source."
```

Reply:
> 

was extra making it short cause long msg are hardly readable but thats also good.

Reply:
ok